src/main/java/org/sagebionetworks/bridge/services/FileService.java

```diff
@@ -158,7 +158,11 @@ public void deleteAllAppFiles(String appId) {
         checkNotNull(guid);
         checkNotNull(createdOn);
 
-        return fileRevisionDao.getFileRevision(guid, createdOn);
+        Optional<FileRevision> opt = fileRevisionDao.getFileRevision(guid, createdOn);
+        if (opt.isPresent()) {
+            opt.get().setDownloadURL(getDownloadURL(opt.get()));
+        }
```

Comment:
Do we have a unit test for this test case?

Reply:
Added case where optional is empty.